content/master/concepts/managed-resources.md

```diff
@@ -211,37 +211,111 @@ resource object is deleted from Kubernetes and the `deletionPolicy` is
+(`Observe`, `Create`, `Update`, `Delete`, `LateInitialize`, `*`), and there are
+multiple combinations that Crossplane supports based on the use case. For 
+example, setting `["Observe"]` makes Crossplane treat the resource as an
+`ObserveOnly` resource, importing the external resources not originally created
+by Crossplane. This allows new managed resources to reference the `ObserveOnly`
+resource, for example, a shared database or network. 
+The `["Observe"]` policy can also place existing resources under the control of
 Crossplane.  
 
 {{< hint "tip" >}}
 Read the [Import Existing Resources]({{<ref "/knowledge-base/guides/import-existing-resources" >}}) 
 guide for more
-information on using the `managementPolicy` to import existing resources.
+information on using the `managementPolicies` to import existing resources.
 {{< /hint >}}
 
+Furthermore, granular `managementPolicies` can be used to skip the late 
+initialization of the managed resource, by not including `LateInitialize` in the
+`managementPolicies`, which is useful when there are conflicting fields between
+the managed resource and the external resource, due to some external resource
+field being controlled externally. 
+
+{{< hint "tip" >}}
+Read the [initProvider]({{<ref "./managed-resources#initprovider" >}}) section
+for more info about how we can use `LateInitialize` and `initProvider` together
+to ignore changes to fields that are controlled externally.
+{{< /hint >}}
+
+The `"Delete"` action replaces the [deletionPolicy]({{<ref "./managed-resources#deletionpolicy" >}})
+field in the managed resource. For now the `deletionPolicy` is still supported,
+but only if it's set to a non default value, which is `Orphan`. If it's set to
+`Orphan`, and the `managementPolicies` is set to `["*"]`, which is default,
+then the external resource will be orphaned when the managed resource is
+deleted. In other cases, non default `managementPolicies` take precedence over
+the `deletionPolicy` field.
+
 #### Options
-* `managementPolicy: FullControl` - **Default** - Crossplane can create, change
-  and delete the managed resource. 
-* `managementPolicy: ObserveOnly` - Crossplane only imports the details of the
-  external resource, but doesn't make any changes to the managed resource. 
+* `*` - ** Default ** - Crossplane can observe, create, change, delete the 
+   external resource and update the managed resource object with late init
+   parameters.
+* `Observe` - managed resource `status.atProvider` will be updated with the 
+   external resource state.
+* `Create` - external resource will be created using the managed resource
+   `spec.initProvider` and `spec.forProvider`.
+* `Update` - external resource will be updated using the managed resource
+   `spec.forProvider`.
+* `Delete` - the external resource will be deleted when the managed resource is
+   deleted.
+* `LateInitialize` - select fields of the managed resource 
```

Comment:
What about something like? 

> * `LateInitialize` - Unprovided spec fields are late-initialized to `spec.forProvider` with the default values from the cloud provider. This enables XP to take full control of the external resource, even those values not provided by the user upfront.

Again a link to a more detailed section would be helpful.